app/src/main/java/com/google/droidjump/models/Animative.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.droidjump.models;
+
+/**
+ * Represents the common behaviour of objects that must animate
+ */
+public interface Animative {
```

Comment:
Since this interface is only about animation of two bitmaps, maybe we could name it accordingly? TwoStepAnimative?

Reply:
I can change but I think there will not be another type of Animative interface(like ThreeStepAnimative or smth like this) thats why I named it like this.

Reply:
Renamed.